content/docs/use-cases/datasets-cache.md

```diff
@@ -0,0 +1,107 @@
+# Datasets Cache
+
+Datasets used in data science tend to exceed typical storage and computing
```

Comment:
Should we limit to data science? How is computing capacity relevant (maybe network capacity would make more sense)? From the rest of the paragraph, it seems like the problem is not exceeding capacity, but reducing time, cost, and complexity of storing and accessing data.

Reply:
> How is computing capacity relevant

One of the scenarios is that a team shares a GPU server. You can plug an external cache shared in local envs with the training env too. That was my thought process but I can remove it from here and just leave the scenario bullet if it's distracting.

Reply:
> maybe network capacity would make more sense

Good idea, I'll use that for now.

> seems like the problem is not exceeding capacity, but reducing time, cost, and complexity

I reordered the sentences to make it more logical.

Reply:
Okay, computing capacity makes more sense once you explain it, so up to you.

Reply:
I like the networking idea better. Computing is implied in that scenario bullet about the GPU server. Thanks!